src/components/UseLater/UploadImageUrl.js

```diff
@@ -1,62 +0,0 @@
-import React, { Component } from "react";

```

Comment:
I don't think we need this file anymore, as the create apartment function can be done on 'src\components\images_uploading\UploadImageFile.js'